### ISSUE TITLE
Running the built cli apps raise an exception

### ISSUE THREAD

**Issue by carlosperate** [carlosperate/ubittool#8]
A missing package, maybe it needs and updated PyInstaller or hook.

Windows:

```
> ubittool-cli.exe --help
Traceback (most recent call last):
  File "C:\Users\appveyor\.virtualenvs\ubitextract-FgtoUL4O\Lib\site-packages\PyInstaller\loader\rthooks\pyi_rth_pkgres.py", line 13, in <module>
    import pkg_resources as res
  File "c:\users\appveyor\.virtualenvs\ubitextract-fgtoul4o\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 623, in exec_module
    exec(bytecode, module.__dict__)
  File "site-packages\pkg_resources\__init__.py", line 86, in <module>
ModuleNotFoundError: No module named 'pkg_resources.py2_warn'
[1916] Failed to execute script pyi_rth_pkgres
```

macOS:
```
$ ./ubittool-cli --help
Traceback (most recent call last):
  File "PyInstaller/loader/rthooks/pyi_rth_pkgres.py", line 13, in <module>
  File "/Users/travis/.local/share/virtualenvs/ubittool-PRs-Fujz/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 623, in exec_module
  File "pkg_resources/__init__.py", line 86, in <module>
ModuleNotFoundError: No module named 'pkg_resources.py2_warn'
[73645] Failed to execute script pyi_rth_pkgres
```

**Comment by carlosperate** [comment]
Temp fix in 43a4e641b2c948dd6781ef8329da15fda44898dd.

Will need to remove it when the next version of PyInstaller is released (anything after v3.6).

**Comment by carlosperate** [comment]
This was resolved when moving to GH Actions.
